libnd4j/include/config.h

```diff
@@ -0,0 +1,29 @@
+#ifndef LIBND4J_CONFIG_H
+#include <execution/Engine.h>
+
+#define LIBND4J_CONFIG_H
+
+#define HAVE_ONEDNN
+
+
+
+#define SD_LIBRARY_NAME "nd4jcpu"
+
+
+/* #undef HAVE_ARMCOMPUTE */
+
+/* #undef ONEDNN_PATH */
+
+/* #undef HAVE_OPENBLAS */
+
+#define OPENBLAS_PATH "/home/agibsonccc/.javacpp/cache/openblas-0.3.19-1.5.7-linux-x86_64.jar/org/bytedeco/openblas/linux-x86_64"
```

Comment:
I assume this file shouldn't have made it into the commit, given that there is a `config.h.in` and that this includes your specific build environment paths.